docs/features.md

```diff
@@ -206,19 +212,20 @@ For more information about the `machete` file, look at the [reference blog post]
 
 On the left side bar you can find other actions (from top to bottom):
 - ![](left_bar_actions/refresh.png) **Refresh Status** &mdash; refresh the graph displayed in main plugin window
-- ![](left_bar_actions/toggle_listing_commits.png) **Toggle Listing Commits** &mdash; show or hide commits belonging to branches (for more details see `Toggle listing commits` section)
-- ![](left_bar_actions/open_machete_file.png) **Open Machete File** &mdash; open the `machete` file in IntelliJ editor (to see what this editor can do see `Edit machete file` section)
-- ![](left_bar_actions/fetch_all_remotes.png) **Fetch All Remotes** &mdash; equivalent to `git fetch --all` command
+- ![](left_bar_actions/toggleListingCommits.png) **Toggle Listing Commits** &mdash; show or hide commits belonging to branches (for more details see `Toggle listing commits` section)
```

Comment:
Can these images be just symlinks to the original images in `frontendIcons`? 🤔 although I'm afraid GitHub might not be able to render them correctly then

Reply:
It's different format (SVG vs PNG). For now I would rather stick to present solution (for the sake of quick release). I will create an issue to investigate and solve that redundancy.